tox.ini

```diff
@@ -1,14 +1,17 @@
 [tox]
-envlist = py3
+envlist = py36, py37
```

Comment:
I don't think this is desirable: I use tox only to ease running tests and packaging. So keeping just py3 allows to use the python version present on the system. On development systems, we use the default python3 installation, on CI, the version used depends on the travis environment, i.e. tested on both 3.6 and 3.7 with your changes.